autoload/airline/extensions/ale.vim

```diff
@@ -8,6 +8,25 @@ function! s:airline_ale_count(cnt, symbol)
   return a:cnt ? a:symbol. a:cnt : ''
 endfunction
 
+function! s:airline_ale_get_line_number(cnt, type) abort
+  if a:cnt == 0
+    return ''
+  endif
+
+  let l:open_lnum_symbol = get(g:, 'airline#extensions#ale#open_lnum_symbol', '(L')
+  let l:close_lnum_symbol = get(g:, 'airline#extensions#ale#close_lnum_symbol', ')')
+  let l:problem_type = (a:type ==# 'error') ? 'E' : 'W'
```

Comment:
The parentheses before `?` are redundant here.

Reply:
I have no problems with the parenthesis and like them for being more obvious. However I personally hate the `l:` variable prefix.